pkg/volume/flexvolume/probe_test.go

```diff
@@ -169,49 +164,18 @@ func TestProberMultipleEvents(t *testing.T) {
 	}
 
 	// Act
-	updated, _, err := prober.Probe()
+	events, err := prober.Probe()
 
 	// Assert
-	assert.True(t, updated)
+	assert.Equal(t, 2, len(events))
```

Comment:
Shouldn't it be 1, since it's the same driver?

Reply:
`initTestEnvironment(t)` create one driver